power-pay-backend/src/test/java/com/adorsys/gis/powerpay/powerpaybackend/PowerPayBackendHealthTest.java

```diff
@@ -0,0 +1,26 @@
+package com.adorsys.gis.powerpay.powerpaybackend;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+@ContextConfiguration(classes = { PowerPayBackendApplication.class })
+public class PowerPayBackendHealthTest {
```

Comment:
This is testing the best case. You should also test the case the server is not sending 200 ok